### PR TITLE
core(runner): fix missing timing properties in the browser

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
PerformanceEntries in Lightrider (and the extension) had no names and no entryTypes.  This manifested as a 0 `timing.total` in the API as well.  

@brendankenny found that browser PerformanceEntry objects do not spread, but in Node they do b/c of the marky lib.  So, simple fix to explicitly call out the `name` and `entryType` fields when parsing the entries.  This will also populate the missing `lh:runner:run` entry to fill in `timing.total`.

**Related Issues/PRs**
fixes: #8638

### CODE REVIEW

File: lighthouse-core/runner.js [hunk 1]
```diff
@@ -186,7 +186,8 @@ class Runner {
     // resolution.
     .map(entry => {
       return /** @type {PerformanceEntry} */ ({
-        ...entry,
+        name: entry.name,
+        entryType: entry.entryType,
         duration: parseFloat(entry.duration.toFixed(2)),
         startTime: parseFloat(entry.startTime.toFixed(2)),
```
Comment (brendankenny) [inline lighthouse-core/runner.js]: can you reorder so it's (`startTime`, `name`, `duration`, `entryType`) just to prevent the churn in `sample_v2.json` :)

Comment (exterkamp) [reply lighthouse-core/runner.js]: We found that at the same time! Done.

**Comment by brendankenny** [comment]
> @brendankenny found that browser PerformanceEntry objects do not spread, but in Node they do b/c of the marky lib

`marky` makes its own `PerformanceEntry` objects, which spread just fine, but the properties on a native `PerformanceEntry` are getters in the prototype chain, so nothing is spread.

Interestingly, running LH in DevTools seems to defeat marky's environment sniffing and so it uses the fallback objects and the entries are ok. The extension is like lr here and only has `{startTime, duration}`

File: lighthouse-core/runner.js [hunk 2]
```diff
@@ -186,9 +186,10 @@ class Runner {
     // resolution.
     .map(entry => {
       return /** @type {PerformanceEntry} */ ({
-        ...entry,
-        duration: parseFloat(entry.duration.toFixed(2)),
         startTime: parseFloat(entry.startTime.toFixed(2)),
+        name: entry.name,
```
Comment (patrickhulce) [inline lighthouse-core/runner.js]: what to add a comment with link to this issue on why we can't spread? :)

**Comment by Hoten** [comment]
can we add a hexa test to LR

**Pull request merged**